pennylane_cirq/cirq_device.py

```diff
@@ -200,7 +200,12 @@ def to_paulistring(self, observable):
         cirq_op = self._observable_map[observable.name]
         if cirq_op is None:
             raise NotImplementedError(f"{observable.name} is not currently supported.")
-        cirq_op.parametrize(*observable.parameters)
+        parameters = (
+            [len(observable.wires)]
```

Comment:
Why are the parameteres `[len(I.wires)]` in the case of an identity?

Reply:
That's just how `cirq.IdentityGate` is defined.

Reply:
For more context, in the cirq plugin, the gates must be parameterized first then applied on the different wires. Identity isn't technically a parameterized gate, but unlike other non parameterized gates which can be directly applied because they all act on a fixed number of wires, the Identity gate must be first initialized with the number of wires it expects.

Reply:
Thanks!